supervisor/bootstrap.py

```diff
@@ -270,3 +282,54 @@ def supervisor_debugger(coresys: CoreSys) -> None:
     if coresys.config.debug_block:
         _LOGGER.info("Wait until debugger is attached")
         ptvsd.wait_for_attach()
+
+
+def setup_diagnostics(coresys: CoreSys) -> None:
+    """Sentry diagnostic backend."""
+
+    def filter_data(event, hint):
+        # modify event here
+        if not coresys.config.diagnostics:
+            return None
+
+        # Not full startup - missing information
+        if coresys.core.state in (CoreStates.INITIALIZE, CoreStates.SETUP):
+            return event
+
+        # Update information
+        with sentry_sdk.configure_scope() as scope:
+            scope.set_context(
+                "home_assistant",
```

Comment:
Should this be context of `supervisor` ? 
```suggestion
                "supervisor",
```